app/routes/analysis/analysis.py

```diff
@@ -93,27 +102,46 @@ async def _zonal_statics(
         "end_date": end_date,
     }
 
-    response_payload_encoded = await _invoke_lambda(payload)
-    response_payload = json.loads(response_payload_encoded.decode())
+    try:
+        response = await _invoke_lambda(payload)
+    except httpx.TimeoutException:
+        raise HTTPException(500, "Query took too long to process.")
 
-    if response_payload["statusCode"] != 200:
+    resp_dict = response.json()
+
+    if response.status_code != 200 or resp_dict.get("statusCode") != 200:
         logger.error(
-            f"Raster analysis lambda returned status code {response_payload['statusCode']}"
+            f"Raster analysis lambda experienced an error. Full response: {resp_dict}"
         )
         raise HTTPException(
             500, "Raster analysis geoprocessor experienced an error. See logs."
         )
 
-    response_data = response_payload["body"]["data"]
+    response_data = response.json()
```

Comment:
https://github.com/wri/gfw-raster-analysis-lambda/blob/master/lambdas/tiled_analysis/src/lambda_function.py#L44

you only want to return `response.json()["body"]["data"]`. See comment above